### PR TITLE
[IMP] Productivity: Discuss Getting Started

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: https://www.odoo.com/web#id=3113715&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

Update Getting Started Discuss page

### CODE REVIEW

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (hojo-odoo) [inline content/applications/productivity/discuss/overview/get_started.rst:36]:
```suggestion
Clicking :guilabel:`Mark as Todo` on a message causes it to appear 
on the :guilabel:`Starred` page, while clicking :guilabel:`Mark as Read` moves the message to :guilabel:`History`.
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (hojo-odoo) [inline content/applications/productivity/discuss/overview/get_started.rst:57]: We're telling the user to select the Discuss app, so it should have a menuselection tag.
```suggestion
To start a chat, go to the :menuselection:`Discuss` app and click on the :guilabel:`+` icon next to :guilabel:`Direct
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (hojo-odoo) [inline content/applications/productivity/discuss/overview/get_started.rst:70]: Add comma
```suggestion
To mention a user within a chat or the chatter, type *@user-name*; to refer to a channel, type
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (hojo-odoo) [inline content/applications/productivity/discuss/overview/get_started.rst:72]: Rewrite
```suggestion
*#channel-name*. The user mentioned will be notified in their *Inbox* or through an email, depending on their communication settings.
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (hojo-odoo) [inline content/applications/productivity/discuss/overview/get_started.rst:87]:
```suggestion
checking their *status*. The status is shown on the left side of a contact’s name on the Discuss
```

**Review by hojo-odoo — COMMENTED** [review]
Hey @tiku-odoo,

I just had a few comments on this. In terms of things that need to be updated throughout the doc, make sure to italicize the Discuss app any time it's not inside of a menuselection tag.

Thanks,
hojo

**Review by hojo-odoo — COMMENTED** [review]
Hey @tiku-odoo,

I just had a few comments on this. In terms of things that need to be updated throughout the doc, make sure to italicize the Discuss app any time it's not inside of a menuselection tag.

Thanks,
hojo

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (tiku-odoo) [reply content/applications/productivity/discuss/overview/get_started.rst:29]: Noted; I've changed all gui labels to represent the label on screen.

**Comment by tiku-odoo** [comment]
@samueljlieber This document is ready for 1st round of technical review. Thanks for your help on this. 

:+1:

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:61]: Icon formatting– add the identifier in parenthesis please :)
```suggestion
To start a chat, go to the :menuselection:`Discuss` app and click on the :guilabel:`+ (plus)` icon
next to :guilabel:`Direct Messages` or :guilabel:`Channels` in the left menu of the dashboard.
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:52]: This image is too wide at 944px, can you please resize it to be 768px wide (933px max), also the image filename contains an underscore, please change it to hyphens

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:63]: underscores in filename here too, please update to hyphens :)

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:68]: please update to use global relative path:
```suggestion
A company can also easily create :doc:`public and private channels
</applications/productivity/discuss/overview/team_communication>`.
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:75]: User input should use literal formatting not italics, and _inbox_ should be lowercase since the text is not in a guilabel
```suggestion
To mention a user within a chat or the chatter, type `@user-name`; to refer to a channel, type
`#channel-name`. The user mentioned will be notified in their *inbox* or through an email, depending
on their communication settings.
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:77]: Image filename- update to use hyphens please :) and add a period to the alt tag

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:84]: Apostrophe fix and no need for italics on employees
```suggestion
   task's followers, and secondly on employees. If the record being searched does not match with
   either a follower or employee, the scope of the search becomes all partners.
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:91]: Apostrophe fix, can use guilabel on Discuss since it is displayed on the UI in this instruction, lowercase messaging menu
```suggestion
checking their *status*. The status is shown on the left side of a contact's name on the
:guilabel:`Discuss` sidebar, on the *messaging menu* and when listed in the *chatter*.
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:105]:
```
   - :doc:`/applications/productivity/discuss/overview/team_communication`
   - :doc:`/applications/productivity/discuss/overview/plan_activities`
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @tiku-odoo! Nice job with the improvements to this doc! I have a few technical corrections and some suggestions for improvement. Please see below and let me know if you have any questions!

Regarding the curly apostrophes, there is a difference between this `’` and this `'` apostrophe. Google Docs seems to automatically switch them, so they need to be converted back to normal apostrophes (`'`). OR follow these instructions for disabling smart quotes in Google Docs to avoid curly apostrophes:
1. Open your Google Docs document.
2. Click on "File" in the menu at the top left corner.
3. Select "Preferences" from the dropdown menu. A pop-up window will appear.
4. In the pop-up window, locate the "Smart quotes" option. It is usually under the "General" tab.
5. Uncheck the box next to "Smart quotes" or any similar option that refers to curly quotes or smart typography.
6. Click "OK" to save the changes and close the Preferences window.
7. Now, you can copy the desired text from Google Docs and paste it into Visual Studio Code without the curly apostrophes being introduced.

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (tiku-odoo) [reply content/applications/productivity/discuss/overview/get_started.rst:25]: Nice catch, thanks for suggesting!

**Comment by tiku-odoo** [comment]
@samueljlieber This document is ready for a last look when you have a moment. I've made all the requested changes and edits to the images/file names. :+1: 

Thanks for your help on these!

File: content/applications/productivity/discuss/overview/team_communication.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/team_communication.rst:13]: Typo
```suggestion
A *public* channel can be seen by everyone, while a *private* one is only visible to users invited
to it.
```

File: content/applications/productivity/discuss/overview/team_communication.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/team_communication.rst:15]: Update filename please
```suggestion
.. image:: team_communication/create-channel.png
```

Also please update the alt tag for this image:

>    :alt: View of the Discuss sidebar and a channel being created in Odoo Discuss.

File: content/applications/productivity/discuss/overview/team_communication.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/team_communication.rst:25]: Line 22-23 Early line break (I cant make a suggestion there)

```
   announcements), whereas a private channel could be used whenever information should be limited to
   specific groups (such as a specific department).
```

File: content/applications/productivity/discuss/overview/team_communication.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/team_communication.rst:29]: Icon formatting
```suggestion
A channel's name, description, email alias, and privacy can be configured by clicking on the channel
settings :guilabel:`⚙️ (gear)` icon on the sidebar.
```

File: content/applications/productivity/discuss/overview/team_communication.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/team_communication.rst:35]: I _think_ members should be lowercase here
```suggestion
Privacy and members
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @tiku-odoo 👋 Thanks for updating `get_started.rst` 🙂 looking good except for a small change– please remove the unused image `handle-by-emails.png` from `get_started/`.

I missed `team_communication.rst` on my first review, please see my technical suggestions below, ultimately if we're already improving a doc we may as well correct the rst. I was unable to add suggestions to the rest of the doc after line 44, so please continue to make the corrections regarding italics, guilabels, image file names, periods on alt tags, and menuselection use.

Thank you for your continued hard work here 🙏

**Comment by tiku-odoo** [comment]
> Hi @tiku-odoo wave Thanks for updating `get_started.rst` slightly_smiling_face looking good except for a small change– please remove the unused image `handle-by-emails.png` from `get_started/`.
> 
> I missed `team_communication.rst` on my first review, please see my technical suggestions below, ultimately if we're already improving a doc we may as well correct all of the rst. I was unable to add suggestions to the rest of the doc after line 44, so please continue to make the corrections regarding italics, guilabels, image file names, periods on alt tags, and menuselection use.
> 
> Also please perform a [search and replace](https://learn.microsoft.com/en-us/visualstudio/ide/finding-and-replacing-text?view=vs-2022) on this doc to convert the curly apostrophes to straight. In VSCode press `CTRL + SHIFT + F` and search for `’`, then replace with `'`.
> 
> Thank you for your continued hard work here pray

Hi Sam, 

I've reverted the changes in the team_communication.rst file. The changes were unintentional. I've fixed the file and image that was changed. In addition, I have removed the image from the Getting Started Folder. 

Thanks, 
Tim

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:25]: I suppose this would be user input, so its probably best to format it as such
```suggestion
sent with an `@` mention, or a notification is sent for a record that the user follows.
```

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (samueljlieber) [inline content/applications/productivity/discuss/overview/get_started.rst:66]: Add a missing period
```suggestion
   :alt: View of Discuss's panel emphasizing the titles channels and direct messages in Odoo
         Discuss.
```

**Review by samueljlieber — APPROVED** [review]
Hi @tiku-odoo this PR is looking great!! Nice job reverting your edits, looks like `get_started` is the only file changed now 🙂 I have just two small improvements, please see below and make them before tagging for final review. Im approving now!

**Comment by tiku-odoo** [comment]
@StraubCreative  This document is ready for your review when you have a moment. Thanks for your help on this review. 

Tim :+1:

**Comment by tiku-odoo** [comment]
@meng-odoo This doc is ready for your review when you have a moment. 

Thanks. Tim :)

**Comment by tiku-odoo** [comment]
CC @meng-odoo This doc is ready for your review :+1:

**Comment by tiku-odoo** [comment]
@odoo/us-doc-review 

This PR is ready for your review! Thank you :)

**Comment by tiku-odoo** [comment]
@odoo/us-doc-review 

Bumping this document to the top... needs a final review.

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (StraubCreative) [inline content/applications/productivity/discuss/overview/get_started.rst:25]: be more specific
what kind of message, what kind of note, what is an example of a record, etc.

for people who know the software we can more or less piece this together but for people who know nothing about it and want to understand the mechanics of these notifications, fully cover the triggers and instances of how and when they fire.

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @tiku-odoo 

I'd like to see the content here expanded more deeply and with more specific language.

The near 1:1 replacements of 2nd-person and small wording changes don't seem to be adding value to the doc; the new version is only about 10 lines longer than the previous one which suggests the content wasn't added to or expanded in a meaningful way. 

As well it doesn't appear we urgently corrected a mistake that needed a merge right away.

If you'd like, we can merge these changes in right away just to quickly update the language, and then do another PR expanding the content out more?

Lmk what you'd like to do :)

**Comment by tiku-odoo** [comment]
@StraubCreative 
I'd like to push these changes to get the vital info updated and address the rest of the doc at a later time.

I've made the requested changes, all good to go! 

Thanks,
Tim

File: content/applications/productivity/discuss/overview/get_started.rst
Comment (StraubCreative) [inline content/applications/productivity/discuss/overview/get_started.rst:25]:
```suggestion
that the user follows. Something that triggers a notification is changing of the stage (if an email
is configured to be sent, for example if the task is set to :guilabel:`Done`).
```

**Review by StraubCreative — APPROVED** [review]

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**
